### PR TITLE
Update to latest apps upstream

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
This was done using the below commands
```
git remote add polkadot-js git@github.com:polkadot-js/apps.git
git fetch polkadot-js
git checkout alephzero
git merge polkadot-js/master
# resolve many, many conflicts; in a nutshell most of the changes in files we did not touch I merged from upstream; those around in endpoints or page-staking carefully I merged using git mergetool 
```

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRelayRococo.ts [hunk 1]
```diff
@@ -372,198 +371,62 @@ export const testParasRococo: EndpointOption[] = [
...
-    genesisHash: ROCOCO_GENESIS,
-    info: 'rococo',
-    text: t('rpc.rococo', 'Rococo', { ns: 'apps-config' }),
-    providers: {
-      Parity: 'wss://rococo-rpc.polkadot.io',
-      'light client': 'light://substrate-connect/rococo'
-      // OnFinality: 'wss://rococo.api.onfinality.io/public-ws', // After reset, node misses host functions
-      // Pinknode: 'wss://rpc.pinknode.io/rococo/explorer' // After reset, syncs to old chain
-      // 'Ares Protocol': 'wss://rococo.aresprotocol.com' // https://github.com/polkadot-js/apps/issues/5767
+export const testParasRococoCommon: EndpointOption[] = [
```
Comment (deuszx) [inline packages/apps-config/src/endpoints/testingRelayRococo.ts]: What's the rule used here for choosing which networks we want to support from our fork?

Comment (Marcin-Radecki) [reply packages/apps-config/src/endpoints/testingRelayRococo.ts]: I'd say upstream repo is right - so for all config files etc upstream is a golden source, ofc we then append our networks

Comment (deuszx) [reply packages/apps-config/src/endpoints/testingRelayRococo.ts]: Got it. I reviewed it as part of the "resolve merge conflicts" commit and thought that we're removing those ourselves. If it's upstream then I get it.

File: packages/apps/public/locales/en/app-alliance.json [hunk 2]
```diff
@@ -28,10 +30,11 @@
   "members": "members",
   "no": "no",
   "prime voter": "prime voter",
+  "retirting": "retirting",
```
Comment (deuszx) [inline packages/apps/public/locales/en/app-alliance.json:33]: retirting? Did you mean _retiring_ maybe?

Comment (Marcin-Radecki) [reply packages/apps/public/locales/en/app-alliance.json:33]: This was not my change, but upstream one

File: packages/apps-config/src/endpoints/testing.ts [hunk 3]
```diff
@@ -27,15 +27,15 @@ export const testChains: EndpointOption[] = [
     info: 'aleph',
     text: 'Aleph Zero Testnet',
     providers: {
-      'Aleph Zero Foundation': 'wss://ws.test.azero.dev'
+      'Aleph Zero Foundation': 'wss://ws.test.azero.dev',
+      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
```
Comment (maciejnems) [inline packages/apps-config/src/endpoints/testing.ts]: Have we decided to add this endpoint to our wallet? Not sure we verified it.

Comment (Marcin-Radecki) [reply packages/apps-config/src/endpoints/testing.ts]: Removed for now

**Review by maciejnems — APPROVED** [review]

**Review by deuszx — APPROVED** [review]
I've gone through all of the files but it's a really big merge. Will we have some time to "click through" it collectively before releasing?

**Pull request merged**